### PR TITLE
kill SoftwareProductType

### PR DESCRIPTION
**Pull request by nickpell**
Cleaning up some legacy code

### CODE REVIEW

File: corehq/apps/accounting/bootstrap/utils.py [hunk 1]
```diff
@@ -127,9 +126,9 @@ def _ensure_product_rate(monthly_fee, edition, verbose, apps):
     try:
         # TODO - remove after squashing migrations
         SoftwareProduct = apps.get_model('accounting', 'SoftwareProduct')
-        product = SoftwareProduct(name=product_name, product_type=SoftwareProductType.COMMCARE)
+        product = SoftwareProduct(name=product_name, product_type='CommCare')
         if hasattr(product, 'product_type'):
-            product.product_type = SoftwareProductType.COMMCARE
+            product.product_type = 'CommCare'
```
Comment (emord) [inline corehq/apps/accounting/bootstrap/utils.py]: What do lines 130 and 131 do? It looks like product_type should always already be 'CommCare'

File: corehq/apps/accounting/utils.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def fmt_product_rate_dict(product_name, product_rate=None):
             product_rate = SoftwareProductRate.objects.create(name=product_name, is_active=True)
     return {
         'name': product_rate.name,
-        'product_type': SoftwareProductType.COMMCARE,
+        'product_type': 'CommCare',  # TODO - kill product_type
```
Comment (emord) [inline corehq/apps/accounting/utils.py]: What's holding you back from killing this? It (maybe naively) seems like you could just remove all references to this?

**Review by emord — APPROVED** [review]

File: corehq/apps/accounting/bootstrap/utils.py (hunk 1 above)
Comment (nickpell) [reply corehq/apps/accounting/bootstrap/utils.py]: Missed this redundancy - removed the if statement

File: corehq/apps/accounting/utils.py (hunk 2 above)
Comment (nickpell) [reply corehq/apps/accounting/utils.py]: It gets used somewhere in JS and killing it breaks the internal accounting pages.  I tried tracking this down but got nowhere - leaving as an exercise for later.

**Review by emord — APPROVED** [review]

**Pull request merged**